### DISCUSSION TITLE
Manual screenshot update for base branch

### DISCUSSION THREAD

**Discussion by gilons** [percy/cli#972] [Support] [answered]
## The problem

I am currently working on a react-native application and want to use Percy, a visual testing service to catch UI changes in my app.

I don't go the storybook way.

Currently, we use detox for testing, it generates screenshots artifacts which we directly upload to Percy.

But the test comparison always fails right now because we don't have corresponding baseline snapshots for the base branch, develop.

So, I want to be able to upload those baseline snapshots for the default branch manually and change them when I want.

I need help with that.

I have already checked the @percy/cli docs, I only see a way to upload manually but I don't see how to do it for the base branch.

Here is the stack overflow question: https://stackoverflow.com/questions/72739336/how-to-manually-upload-baseline-screenshots-in-percy-for-base-branch

Thanks

## Environment

- Node version:
- `@percy/cli` version: 1.0.8
- Version of Percy SDK you’re using:
- If needed, a build or snapshot ID:
- OS version: MacOS Montery
- Type of shell command-line [interface]:

## Details

If necessary, describe the problem you have been experiencing in more detail.

## Debug logs

If you are reporting a bug, _always_ include logs! [Give the "Debugging SDKs"
document a quick read for how to gather logs](https://docs.percy.io/docs/debugging-sdks#debugging-sdks)

Please do not trim or edit these logs, often times there are hints in the full
logs that help debug what is going on.

## Code to reproduce issue

Given the nature of testing/environment bugs, it’s best to try and isolate the
issue in a reproducible repo. This will make it much easier for us to diagnose
and fix.

**Comment by Robdel12** [comment, marked as answer]
👋🏼 hey @gilons! You can use `PERCY_BRANCH` to change the builds git branch and run a build to upload a set of screenshots you'd like to use for a baseline. https://docs.percy.io/docs/environment-variables#optional Something like:

`PERCY_BRANCH="your-baseline-branch-name-here" npx percy upload ./screenshots` (or whatever the CLI command you're running -- if it's the CLI the `PERCY_BRANCH` env var will work)

**Reply by gilons** [reply]
Thanks @Robdel12  that worked
